proxy-node-gateway/src/main/java/uk/gov/ida/notification/exceptions/mappers/ExceptionToErrorPageMapper.java

```diff
@@ -40,34 +36,9 @@ public Response toResponse(TException exception) {
 
     protected abstract Response.Status getResponseStatus(TException exception);
 
-    String getLogId() {
-        return logId;
-    }
-
-    protected String getAuthnRequestId(TException exception) {
-        return null;
-    }
-
-    protected String getIssuerId(TException exception) {
-        return null;
-    }
-
-    protected String getSessionId(TException exception) {
-        return null;
-    }
-
-    protected DateTime getIssueInstant(TException exception) {
-        return null;
-    }
-
     private void logException(TException exception) {
-        this.logId = String.format("%016x", ThreadLocalRandom.current().nextLong());
-        final String message = exception.getMessage();
-        final String cause = Optional.ofNullable(exception.getCause()).map(Throwable::getMessage).orElse(null);
-
-        LOG.error(format("Error whilst contacting uri [{0}]; logId: {1}; requestId: {2}; sessionId: {3}, issuer: {4}; issueInstant: {5}; message: {6}, cause: {7}",
-                uriInfo.getPath(), logId, getAuthnRequestId(exception), getSessionId(exception), getIssuerId(exception), getIssueInstant(exception), message, cause),
-                exception
-        );
+        proxyNodeLogger.addContext(exception);
+        // This error level will be addressed in the next PR/Story concerning ExceptionToSamlErrorResponseMapper
+        proxyNodeLogger.log(Level.WARNING, format("Error whilst contacting uri [{0}]", this.uriInfo.getPath()));
```

Comment:
What's your plan to deal with this?

Reply:
Not exactly sure at this stage.  At the moment ProxyNodeLogger uses the java.util.logging.Logger which doesn't support ERROR as a level.  We can't just swap out to slf4j style because that uses methods to set the level logged at rather than something being passed in to the log method.  The final plan should involve some discussion across the team as there's no obvious nice solution.